tensorflow/compiler/mlir/lite/quantization/quantization_config.cc

```diff
@@ -48,7 +48,7 @@ bool ParseInputNodeQuantSpecs(absl::string_view node_names,
   std::vector<double> node_mins;
   if (!min_values.empty()) {
     std::vector<std::string> node_mins_str = absl::StrSplit(min_values, ',');
-    for (int i = 0; i < node_mins_str.size(); i++) {
+    for (int i : llvm::seq<int>(node_mins_str.size())) {
```

Comment:
This is a more natural for-range loop here I believe: `for (const std::string&node_min : node_mins_str.size()) {`

Reply:
Is my latest commit 3400536 the change you are suggesting?

Reply:
Yes, in order of preference I would always go with a for-range loop whenever possible:

```
for(const std::string node_min_str : node_mins_str) {
  ...
}
```

Otherwise if I need both the index and the value and I am iterating over an entire container, I would do:

```
for(auto &indexed_str : llvm::enumerate(node_mins_str)) {
  int i = indexed_str.index();
  std:string &node_min_str = indexed_str.value();
  ...
}
```

Finally if you only need the index over a range and not the matching container item:

```
for (int i : llvm::seq<int>(node_mins_str.size())) {
   ...
}
```

And for general condition that won't fix the above:

```
```
for (int i = <init>, e = <end>;  i < e; ++i) {
   ...
}
```
(and choosing a better variable name than `i` as much as possible).

The most general form otherwise (in particular when the "end expr" must be re-evaluated with each iteration):

```
for (int i = <init>;  i <  <end expr>; ++i) {
   ...
}
```